docs/source/tutorials/memory_optimizations.rst

```diff
@@ -232,6 +239,9 @@ through the `CPUOffloadOptimizer <https://github.com/pytorch/ao/tree/main/torcha
 This optimizer can wrap any base optimizer and works by keeping the optimizer states and performing the optimizer step on CPU, thus reducing
 GPU memory usage by the size of the optimizer states. Additionally, we can also offload gradients to the CPU by using `offload_gradients=True`.
 
+If finetuning on a single-device, another option is to use the PagedAdamW8bit from bitsandbytes, mentioned above, which will *only* offload to CPU
```

Comment:
Can you link to `PagedAdamW8bit`, and also the "above" in "from bitsandbytes, mentioned above", please?

Reply:
done